notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>You need to install aiplatform regardless of whether it is ran in Colab. The install and restart (if Colab) should be separate cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>